Jenkinsfile.treecompose

```diff
@@ -81,21 +83,27 @@ node(NODE) {
         // delete the ref head (so rpm-ostree won't add a parent) and
         // then we do a repo prune after.
         def version, commit
-        stage("Compose Tree") { sh """
-            rm ${repo}/refs/heads/* -rf
-            env G_DEBUG=fatal-warnings rpm-ostree compose tree --repo=${repo} \
-                                                        --add-metadata-string=version=${version_prefix}.${env.BUILD_NUMBER} \
-                                                        ${manifest}
-            ostree --repo=${repo} rev-parse ${ref} > commit.txt
-        """
-            commit = readFile('commit.txt').trim();
-            version = utils.get_rev_version("${repo}", commit)
-            currentBuild.description = "🆕 ${version} (commit ${commit})";
-            if (previous_commit.length() > 0) {
-                sh "rpm-ostree --repo=${repo} db diff ${previous_commit} ${commit}"
+        try {
```

Comment:
I don't think it's necessary to wrap this whole stage in a `try{}`, but it shouldn't cause any problems leaving it in.

Reply:
I could be totally wrong since jenkins is (seems to be) unpredictable, but it failed saying it doesn't understand the command `archiveArtifacts` when I had the `finally` statement here-ish: https://github.com/Rahuls0720/rhcos/blob/pkg_diff-jenkinsTESTING/Jenkinsfile#L98

Reply:
I'll try testing it without the try again

Reply:
LOL nvm ^...already merged :)